pkg/storaged/lvol-tabs.jsx

```diff
@@ -107,15 +107,15 @@
             if (size - crypto_overhead > vdo.physical_size)
                 return vdo.grow_physical();
             else if (size - crypto_overhead < vdo.physical_size)
-                return cockpit.reject(_("VDO backing devices can not be made smaller"));
+                return Promise.reject(_("VDO backing devices can not be made smaller"));
             else
-                return cockpit.resolve();
+                return Promise.resolve();
         } else if (size < orig_size) {
             // This shouldn't happen.  But if it does, continuing is harmful, so we throw an error.
-            return cockpit.reject(_("Unrecognized data can not be made smaller here."));
+            return Promise.reject(_("Unrecognized data can not be made smaller here."));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)